SchedulerBot/SchedulerBot.Database.Entities/ScheduledMessageDetailsServiceUrl.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+
+namespace SchedulerBot.Database.Entities
+{
+	/// <summary>
+	/// Enables many-to-many relationship between <see cref="ScheduledMessageDetails"/> and <see cref="ServiceUrl"/>.
+	/// </summary>
+	public class ScheduledMessageDetailsServiceUrl
```

Comment:
Add `ICreatedOn` interface?

Reply:
Did it, although there is no real usage for that for now.